lxd/cluster/membership.go

```diff
@@ -415,9 +415,15 @@ func Join(state *state.State, gateway *Gateway, cert *shared.CertInfo, name stri
 			return errors.Wrapf(err, "failed to unmark the node as pending")
 		}
 
-		// Attempt to send a heartbeat to all other nodes
-		for _, node := range nodes {
-			go heartbeatNode(context.Background(), node.Address, cert, nodes)
+		// Send partial heartbeat to raft nodes containing just a raft node list.
```

Comment:
This comment seems odd as nothing is sent at this stage, maybe the later comment (just above the loop) needs to be removed so this becomes the main comment for the block?

Reply:
fixed